### ISSUE TITLE
Nav Bar Information Architecture

### ISSUE THREAD

**Issue by carynaudenried** [bethlakshmi/gbe-divio-djangocms-python2.7#501]
Review the Menu bar and sub-menus to organize information and content.

**Comment by burlexpo** [comment]
@carynaudenried do we also think that when performer applications are open, it's worth it to have an "Apply" button up top near the "Volunteer" button?

**Comment by carynaudenried** [comment]
@burlexpo this looks good! I agree with the idea to add a "Volunteer" button and "Apply" button up top to call out those actions, when they are applicable. It'd be good to remove the Apply when applications close!

I took your first pass and made some adjustments. Let me know what you think.
https://docs.google.com/spreadsheets/d/1zTIyCecM7OqxOAbfwNrpJfQ4pwK35AOKMtKJnC1axxY/edit?usp=sharing

My thoughts:
- "About Us" is often the last menu to the right on websites, I think it'd be good to place there so people know where to look. I renamed from "Essentials" as that doesn't really describe what's in it. I'd be worried people wouldn't know what to look for in that menu.
- I moved "Tickets" into the Shows menu. That plus a big callout on the homepage will be bold enough for anyone to be able to find, esp considering people coming to the website to buy tickets will land on the homepage first -- we'll make it super obvious
